.github/workflows/ci.yml

```diff
@@ -37,6 +37,30 @@ jobs:
             -v nominatim-postgres:/var/lib/postgresql/12/main \
             -p 8081:8080 \
             nominatim &
+          sleep 30
+          ./assert-non-empty-json "http://localhost:8081/search.php?q=avenue%20pasteur"
+
+      - name: Check import with bind-mount
+        working-directory: .github/workflows
+        run: |-
+          docker run -i --rm \
+            -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf \
+            -e REPLICATION_URL=http://download.geofabrik.de/europe/monaco-updates/ \
+            -v /tmp/nominatim-postgres:/var/lib/postgresql/12/main \
+            -p 8081:8080 \
+            nominatim &
+          sleep 30
+          ./assert-non-empty-json "http://localhost:8081/search.php?q=avenue%20pasteur"
+
+      - name: Check import with bind-mount
+        working-directory: .github/workflows
+        run: |-
+          docker run -i --rm \
+            -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf \
+            -e REPLICATION_URL=http://download.geofabrik.de/europe/monaco-updates/ \
+            -v /tmp/nominatim-postgres:/var/lib/postgresql/12/main \
+            -p 8081:8080 \
```

Comment:
Port 8081 is already used by the container from the previous step so you want to use another one.

One day I will make the CI more efficient by running the various checks in parallel in different jobs and then this cannot happen anymore.

Reply:
Thanks, I've overlooked that one. Tried 30 seconds, seemed to short.